var/spack/repos/builtin/packages/rocprofiler-dev-api/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RocprofilerDevApi(Package):
```

Comment:
You're not doing anything with rocprofiler during install, can you please turn this into roctracer-dev-api?